<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget_parameters">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Properties</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_Input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLabel" name="label_InputName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Input">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_FileStream">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>File Stream</string>
    </property>
    <widget class="QPushButton" name="pushButton_create_File">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>create File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_File">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>load File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filename:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FileName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RightInstance1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Plot">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Plot</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_maxIterations">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxIterations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max. Iterations:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_plot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_range">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>range Check</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Fitness">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_plot</tabstop>
  <tabstop>pushButton_load_File</tabstop>
  <tabstop>lineEdit_FileName</tabstop>
  <tabstop>treeWidget_parameters</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
